setupaliases.lic

```diff
@@ -16,6 +16,15 @@
 
   # toggles training-manager on or off
   ['tm', "#{$clean_lich_char}eq Script.running?('training-manager') ? kill_script('training-manager') : start_script('training-manager')"],
+  
```

Comment:
```suggestion
 
```

Reply:
Line 19 is a space. Not just a newline/carriage return.

Reply:
Ah, when I went back to my repo in the web editor, I must have been on the wrong branch.  Thanks!

Reply:
All good. :)